app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart

```diff
@@ -99,7 +99,7 @@ class PostsubmitLuciSubscription extends SubscriptionHandler {
     }
 
     // Only update GitHub checks if target is not bringup
-    if (target.value.bringup == false) {
+    if (target.value.bringup == false && config.postsubmitSupportedRepos.contains(target.slug)) {
```

Comment:
where is it restricting to `packages` only?

Reply:
https://github.com/flutter/cocoon/blob/main/app_dart/lib/src/service/config.dart#L61

Reply:
There are only two supported repos in that postsubmitSupportedRepos list.

Reply:
Then is this wrong? description says to limit to packages only.

Reply:
Now with `plugins` repo merged to `packages` repo, the `postsubmitSupportedRepos` is actually only `packages`. We need a clean up. Created https://github.com/flutter/flutter/issues/122334 to track.

Also updated title to avoid confusion.